src/generators/web.cr

```diff
@@ -66,8 +66,6 @@ class LuckyCli::Generators::Web
 
   private def setup_gitignore
     append_text to: ".gitignore", text: <<-TEXT
-    start_server
-    *.dwarf
```

Comment:
I think we can actually just keep this. In my apps, I can't run `lucky dev` (technical reasons), so I actually just run the build myself and if I forget to send it to the bin, it'll leave a .dwarf file.